lib/generators/ror_push_notifications/migrations_generator.rb

```diff
@@ -15,22 +15,18 @@ def self.base_root
       source_root File.expand_path('../templates/migrations', __FILE__)
 
       def create_migrations
-=begin
-        Dir.glob(File.join(self.class.base_root, 'templates', 'migrations', '*.rb')).sort.each do |file|
-          migration_template("#{File.basename(file)}", "db/migrate/#{File.basename(file)}")
-        end
-=end
-
         templates = %w(create_rpn_configs create_rpn_devices create_rpn_notifications)
 
         templates.each do |file|
-          begin
-            migration_template("#{file}.rb", "db/migrate/#{file}.rb")
-          rescue => err
-            puts "WARNING: #{err.message}"
-          end
+          migration_template("#{file}.rb", "db/migrate/#{file}.rb")
         end
       end
+
+      def self.next_migration_number(path)
```

Comment:
Unused method argument - `path`. If it's necessary, use `_` or `_path` as an argument name to indicate that it won't be used. You can also write as `next_migration_number(*)` if you want the method to accept any arguments but don't care about them.